interfaces/cos_agent/v0/schemas/provider.json

```diff
@@ -0,0 +1,267 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://canonical.github.io/charm-relation-interfaces/cos_agent/schemas/provider.json",
+    "type": "object",
+    "title": "`cos_agent` provider root schema",
+    "description": "The `cos_agent` root schema comprises the entire provider databag for this interface.",
+    "default": {},
+    "examples": [
+        {
+        "application-data": {
+            "config": {
+                "metrics": {
+                    "alert_rules": {
+                        "groups": [
+                            {
+                            "name": "test_58b48ff0_zookeeper_cos_23790144_zinc",
+                            "rules": [
+                                {
+                                "alert": "ZincTargetMissing",
+                                "annotations": {
+                                    "description": "A Prometheus target has disappeared. An exporter might be crashed.\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}",
+                                    "summary": "Prometheus target missing (instance {{ $labels.instance}})"
+                                },
+                                "expr": "up{juju_application=\"zinc2\"} == 0",
+                                "for": "0m",
+                                "labels": {
+                                    "app_group": "cloud",
+                                    "juju_application": "zookeeper",
+                                    "juju_charm": "zookeeper",
+                                    "juju_model": "test",
+                                    "juju_model_uuid": "58b48ff0-608b-435f-83b9-ea643e0b98db",
+                                    "severity": "critical"
+                                }
+                            }
+                            ],
+                        },
+                        ],
+                    }
+                },
+                "logs": {
+                    "alert_rules": {}
+                },
+                "dashboards": {
+                    "dashboards": [
+                        "/Td6WFoAAATm1rRGAgAhARYAAAB0L+Wj6hvVPXVdA…",
+                        "/Td6WFoAAATm1rRGAgAhARYAAAB0L+Wj6hvVPXVdB…",
+                    ],
+                },
+            },
+        },
+        "unit-data": {
```

Comment:
This is not how relation data is modeled on the unit-level. See https://github.com/canonical/charm-relation-interfaces/blob/c9ca53c0f3b9cb8933719279ea5bf12131f4f383/interfaces/prometheus_scrape/v0/schemas/provider.json#L238-L245 for an example